app/views/forms/_edit.html.slim

```diff
@@ -0,0 +1,4 @@
+ul.list-group.sortable
```

Comment:
minor improvement - would be cool if the form editor page said the name of the form you're editing.